app/controllers/katello/api/v2/content_view_versions_controller.rb

```diff
@@ -84,31 +84,35 @@ def republish_repositories
 
     api :POST, "/content_view_versions/:id/export", N_("Export a content view version"), :deprecated => true
     param :id, :number, :desc => N_("Content view version identifier"), :required => true
-    param :export_to_iso, :bool, :desc => N_("Export to ISO format"), :required => false
-    param :iso_mb_size, :number, :desc => N_("maximum size of each ISO in MB"), :required => false
-    param :since, Date, :desc => N_("Optional date of last export (ex: 2010-01-01T12:00:00Z)"), :required => false
+    param :export_to_iso, :bool, :desc => N_("Export to ISO format, relevant only on Pulp 2 repositories"), :required => false
```

Comment:
In general, if our users are knowing about the difference between pulp3 and pulp2 we've failed. I see what you're trying to do here, though. I would be fine with removing this or logging a warning if it's used against the wrong implementation. If you wanna keep it, it should be two sentences :)

Reply:
Alternately I can raise errors on unsupported parameters if you prefer ? So if your yum is pulp3 and you passed in pulp2 params I can complain. Would you like that?

Reply:
That sounds good to me!

Reply:
Do you plan to update per above?

Reply:
updated